languages/scala/reference/README.md

```diff
@@ -0,0 +1,211 @@
+# Scala reference
+
+## Concepts
+
+The Scala concept exercises are based on concepts. The list below contains the concepts that have been identified for the Scala language.
+
+### Language-unique
+
+- Implicits
+- path-dependent-path
+
+### Functional
+
+- [Functions][functions]
+  - [Pure functions][pure_functions]
+  - [Higher-order functions][higher_order_functions]
+  - [Anonymous functions][anonymous_functions]
+  - [Nested functions][nested_functions]
+  - [Function composition][function_composition]
+  - [Partial application][partial_application]
+  - [Recursion][recursion]
+  - [Partial function][partial_function]
+    - Tail recursion
```

Comment:
I believe `Tail recursion` should be part of `Recursion`, not part of `Partial function`

Reply:
oups, thanks for catching !